runNode.sh

```diff
@@ -62,10 +62,24 @@ pk=adbacd10fdb9822c71025d6d00092b8a4abb5ebcb673d28d863f7c7c5adaddf3
 # Launch session
 s="d-voting-test"
 
-tmux list-sessions | rg "^$s:" >/dev/null 2>&1 && { echo >&2 "A session with the name $s already exists; kill it and try again"; exit 1; }
+# check if session already exists, if so run the kill_test.sh script
+if tmux has-session -t $s 2>/dev/null; then
+  echo "Session $s already exists, killing it"
+  ./kill_test.sh
+fi
+
+#tmux list-sessions | rg "^$s:" >/dev/null 2>&1 && { echo >&2 "A session with the name $s already exists; kill it and try again"; exit 1; }
+ 
+
 
 tmux new-session -d -s $s
 
+# display warning if the number of nodes is less or equal than 3 because it doesn't respect the Byzantine Fault Tolerant
```

Comment:
I don't find the error message, nor the comment, very informative.
Say something like:

```
# Checks that we can afford to have at least one Byzantine node and keep the
# system working, which is not possible with less than 4 nodes.
```